x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/build_alert_group_from_sequence.test.ts

```diff
@@ -134,4 +138,342 @@ describe('buildAlert', () => {
       expect(groupId).toEqual(groupIds[0]);
     }
   });
+
+  describe('recursive intersection between objects', () => {
```

Comment:
These tests got lost in the migration - adding them back since "recursively merging objects" is a bit tricky.